video/out/win32/menu.c

```diff
@@ -218,6 +222,17 @@ void mp_win32_menu_update(struct menu_ctx *ctx, struct mpv_node *data)
     talloc_free_children(ctx->ta_data);
 
     build_menu(ctx->ta_data, ctx->menu, data);
+    if (!ctx->updated) {
```

Comment:
What about making a copy of ctx->menu for system menu here to make it easier:

1. always reset system menu
2. check menu item count, if >0 then append submenu.